aggregator/fetchers/factory.go

```diff
@@ -57,6 +66,12 @@ func createFetcher(fetcherName string, responseGetter aggregator.ResponseGetter)
 			responseGetter,
 			baseFetcher{},
 		}, nil
+	case MaiarName:
+		return &maiar{
+			responseGetter,
```

Comment:
I think is a good idea to change all these structs instantiations to provide also the field name:
```
return &maiar{
    responseGetter,
    baseFetcher{},
    maiarTokensMap,
}
```

becomes

```
return &maiar{
    ResponseGetter: responseGetter,
    baseFetcher:    baseFetcher{},
    maiarTokensMap: maiarTokensMap,
}
```

valid for all structs